ee/fleet-agent-downloader/api/controllers/entrance/signup-sso-user-or-redirect.js

```
@@ -0,0 +1,55 @@
+    }
+    // If the requesting user has a session, but it does not contain a passport object, we'll redirect them to the login page.
+    if (!this.req.session.passport || !this.req.session.passport.user) {
+      throw {redirect: '/login'};
+    }
+
+    let ssoUserInfo = this.req.session.passport.user.userinfo;
+    if (!ssoUserInfo || !ssoUserInfo.email || !ssoUserInfo.name) {
+      sails.log.warn('SSO login missing required userinfo claims (email/name).');
+      throw {redirect: '/login'};
+    }
+    let possibleUserRecordForThisSsoUser = await User.findOne({emailAddress: ssoUserInfo.email});
```

ESLint

[error] 'possibleUserRecordForThisSsoUser' is never reassigned. Use 'const' instead.

(prefer-const)